### DISCUSSION TITLE
SAE; MQTT auf VM Ubuntu 22.04.4 LTS installieren

### DISCUSSION THREAD

**Discussion by alexb2506** [camueller/SmartApplianceEnabler#613] [Installation und Betrieb]
Hallo zusammen,
ich versuche aktuell SAE auf meiner Synology VMM Ubuntu Server zu installieren.

Mein System ist: Ubuntu 22.04.4 LTS (GNU/Linux 5.15.0-112-generic x86_64)

Aktuell sind folgende Anwendungen bereits installiert:
- Grafana 11
- Node-red 3.10
- Influx 1.8

AKtuell scheiters es daran Mosquitto zu installieren. Und natürlich auch die Instalation von SAE ist auch gescheitert.

Eventuell kann mir jemand mit Befehlen zu Installation beider Apps aushelfen.

Wird auch Java benötigt?
VG Alex

**Comment by camueller** [comment]
Versuche es doch mal mit der Dokumentation - da steht alles drin.

**Comment by alexb2506** [comment]
Hallo Axel,
habe deinem Rat gefolgt und es doch hinbekommen. Jetzt habe ich nur ein Problem: Wie bekomme ich meine vorhadene xml Dateien auf den Server um es auszutauschen?
Gint es einen Webserver den Du mir empfehlen kannst? Oder kann ich ohne weiteres Webmin auf dem Ubuntu installieren?

**Comment by camueller** [comment]
Auch das steht in der Doku: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/ConfigurationFiles_DE.md#-kopieren-der-konfigurationsdateien-zwischen-raspberry-pi-und-pc

Du kannst alternativ auch `webmin` installieren.

**Comment by alexb2506** [comment]
Jetzt habe ich es kurz am laufen gehabt, nun kommen Fehler nach der Installation von webmin.

`× smartapplianceenabler.service - Smart Appliance Enabler
     Loaded: loaded (/lib/systemd/system/smartapplianceenabler.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Fri 2024-06-21 15:16:19 CEST; 13min ago
        CPU: 53.119s

Jun 21 15:15:17 ubuntu-homenas sudo[2381]:     root : PWD=/opt/sae ; USER=sae ; COMMAND=/usr/bin/java -Djava.awt.headless=true -Xmx256m -Duser.language=de -Duser.country=DE -DPIGPIOD_HOST=localhost -Dlogging.config=/opt/sae/logback-spring.xml -Dsae.pidfile=/var/run/sae/smartapplianceenabler.pid -Dsae.home=/opt/sae >
Jun 21 15:15:17 ubuntu-homenas sudo[2381]: pam_unix(sudo:session): session opened for user sae(uid=1001) by (uid=0)
Jun 21 15:15:17 ubuntu-homenas systemd[1]: Dependency failed for Smart Appliance Enabler.
Jun 21 15:15:17 ubuntu-homenas systemd[1]: smartapplianceenabler.service: Job smartapplianceenabler.service/start failed with result 'dependency'.
Jun 21 15:16:19 ubuntu-homenas smartapplianceenabler[2371]: ............................................................
Jun 21 15:16:19 ubuntu-homenas smartapplianceenabler[2371]: Unable to start, see /var/log/smartapplianceenabler.log and /var/log/smartapplianceenabler.err
Jun 21 15:16:19 ubuntu-homenas systemd[1]: smartapplianceenabler.service: Control process exited, code=exited, status=1/FAILURE
Jun 21 15:16:19 ubuntu-homenas sudo[2381]: pam_unix(sudo:session): session closed for user sae
Jun 21 15:16:19 ubuntu-homenas systemd[1]: smartapplianceenabler.service: Failed with result 'exit-code'.
Jun 21 15:16:19 ubuntu-homenas systemd[1]: smartapplianceenabler.service: Consumed 53.119s CPU time.
`

und 

`× mosquitto.service - Mosquitto MQTT Broker
     Loaded: loaded (/lib/systemd/system/mosquitto.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Fri 2024-06-21 15:16:35 CEST; 15min ago
       Docs: man:mosquitto.conf(5)
             man:mosquitto(8)
   Main PID: 3071 (code=exited, status=1/FAILURE)
        CPU: 44ms

Jun 21 15:16:35 ubuntu-homenas systemd[1]: mosquitto.service: Scheduled restart job, restart counter is at 5.
Jun 21 15:16:35 ubuntu-homenas systemd[1]: Stopped Mosquitto MQTT Broker.
Jun 21 15:16:35 ubuntu-homenas systemd[1]: mosquitto.service: Start request repeated too quickly.
Jun 21 15:16:35 ubuntu-homenas systemd[1]: mosquitto.service: Failed with result 'exit-code'.
Jun 21 15:16:35 ubuntu-homenas systemd[1]: Failed to start Mosquitto MQTT Broker.
`

Eine Idee woran es liegen kann?
